### PR TITLE
Created help page with FAQs

### PR DESCRIPTION
**Pull request by Sumanbhadra**
A new Help Page to the project, providing a centralized resource for users to find answers and support. The Help Page includes a wide range of frequently asked questions (FAQs) covering various topics related to the company and webpage.
Added contact info for contacting support team.

Issue: #344 

Link added to the footer as instructed earlier.
Styled according to webpage aesthetics and colour pallet.

@RamakrushnaBiswal If any changes are needed, Lemme know. Happy to contribute.

## Summary by CodeRabbit

- **New Features**
	- Introduced a Help and Support page featuring a FAQ section for user inquiries.
	- Added a new navigation link for "Help and Support" in the footer.
	- Implemented a new route for the Help and Support component, accessible at `/help`.

- **Bug Fixes**
	- Minor formatting adjustments made to enhance the layout and readability of the footer component.

### CODE REVIEW

File: frontend/src/components/Pages/HelpAndSupport.jsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+                <IoMdCall />
+                <p>718-971-1684</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+    </>
+  );
+}
```
[the remark this answers is not in the document]
Comment (RamakrushnaBiswal) [reply frontend/src/components/Pages/HelpAndSupport.jsx:142]: @Sumanbhadra add it

**Comment by RamakrushnaBiswal** [comment]
@Sumanbhadra also match the colors with the UI and remove the footer part of the page like emails and phone

**Comment by Sumanbhadra** [comment]
> @Sumanbhadra also match the colors with the UI and remove the footer part of the page like emails and phone

Could you please be specific about the color of which part ?

**Comment by RamakrushnaBiswal** [comment]
@Sumanbhadra like the questions bg
image
this

**Comment by Sumanbhadra** [comment]
@RamakrushnaBiswal Changed the UI color of FAQ to homepage bgcolor and removed the footer (email and phone no) from the page.

Could you please merge it now.

**Review by RamakrushnaBiswal — APPROVED** [review]

**Comment by RamakrushnaBiswal** [comment]
@Sumanbhadra make it responsive and 
change the size 
image

**Comment by Sumanbhadra** [comment]
Is it okay ? @RamakrushnaBiswal

**Comment by RamakrushnaBiswal** [comment]
@Sumanbhadra its good make it responsive

**Comment by Sumanbhadra** [comment]
@RamakrushnaBiswal Done 👍

**Pull request merged**